processor/transformer/transformer.go

```diff
@@ -455,6 +455,7 @@ func (trans *handle) doPost(ctx context.Context, rawJSON []byte, url, stage stri
 				}
 
 				req.Header.Set("Content-Type", "application/json; charset=utf-8")
+				req.Header.Set("X-Feature-Gzip-Support", "?1")
```

Comment:
question: why are we using `?1` as a value?

Reply:
any special meaning or something we come up with it. just curious.